src/mlpack/bindings/python/print_doc_functions_impl.hpp

```diff
@@ -240,7 +240,8 @@ inline std::string ProgramCall(const std::string& programName)
   bool first = true;
   for (auto it = parameters.begin(); it != parameters.end(); ++it)
   {
-    if (!it->second.input || it->second.persistent)
+    if (!it->second.input || (it->second.persistent &&
+        it->second.name != "verbose" && it->second.name != "copy_all_inputs"))
```

Comment:
Ah, nice, thanks for fixing this for Python too.  But `copy_all_inputs` is Python-specific, so we should leave that in. :+1:

Reply:
Removed. :)